cmd/juju/model/defaultscommand.go

```diff
@@ -45,20 +58,41 @@ See also:
 
 // NewDefaultsCommand wraps defaultsCommand with sane model settings.
 func NewDefaultsCommand() cmd.Command {
-	return modelcmd.WrapController(&defaultsCommand{})
+	defaultsCmd := &defaultsCommand{
+		newCloudAPI: func(caller base.APICallCloser) cloudAPI {
+			return cloudapi.NewClient(caller)
```

Comment:
I was nudged to try this different style, because it doesnt' require adding an api member to the defaultsCmd struct. 

I am not sold on it, but tried it here. Perhaps @axw can weigh in since @wallyworld is out.